repositories/memory/topics/get_topic_test.go

```diff
@@ -0,0 +1,98 @@
+package topics
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"testing"
+
+	"github.com/matheusmosca/walrus/domain/entities"
+	"github.com/matheusmosca/walrus/domain/vos"
+	"github.com/stretchr/testify/require"
+)
+
+type args struct {
+	topicName vos.TopicName
+}
+type testCase struct {
+	description string
+	args        args
+	beforeRun   func(storage map[vos.TopicName]entities.Topic)
+	wantErr     error
+}
```

Comment:
It should not be on global scope, you should've create it on each test function

Reply:
Ack. Will make the required changes